src/operators/infinitepepo.jl

```diff
@@ -2,6 +2,17 @@
     struct InfinitePEPO{T<:PEPOTensor}
 
 Represents an infinite projected entangled-pair operator (PEPO) on a 3D cubic lattice.
+
+## Fields
+
+$(TYPEDFIELDS)
+
+## Constructors
+
+    InfinitePEPO(A::AbstractArray{T, 3})
+    InfinitePEPO([f=randn, T=ComplexF64,] Pspaces, Nspaces, Espaces)
+    InfinitePEPO(A::PEPOTensor; unitcell=(1, 1, 1))
+    InfinitePEPO([f=randn, T=ComplexF64,] Pspace::S, Nspace::S, [Espace::S]; unitcell=(1,1,1)) where {S<:ElementarySpace}
```

Comment:
Similar comment about constructors here, If they are documented for the individual functions I don't think we should put them here